MapboxSearch/gradle/versions.gradle

```diff
@@ -41,13 +41,13 @@ ext {
 
     pitest_version = '1.6.7'
 
-    mapbox_maps_version = '11.10.0-rc.1'
+    mapbox_maps_version = '11.10.0'
```

Comment:
Isn't it supposed to be 11.11.0-beta.1?

Reply:
Yes, but it's not available yet. We usually prepare Search and Maps releases in parallel, so the Maps dependency is not ready we release search.

It's not an issue, Search doesn’t depend on the Maps SDK. This dependency is only used in the sample module.